packages/components/src/draggable/README.md

```diff
@@ -49,10 +51,54 @@ const MyDraggable = () => (
 					elementId="draggable-panel"
 					transferData={ { } }
 				>
-					<Dashicon icon="move" />
+				{
+					( { onDraggableStart, onDraggableEnd } ) => (
+						<Dashicon
+							icon="move"
+							onDragStart={ onDraggableStart }
+							onDragEnd={ onDraggableEnd }
+							draggable
+							/>
```

Comment:
Minor: We should probably indent one tab less here

Reply:
bbc83ac